apps/events.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs
+key: events
+title: Events
+---
+
+Events are triggered when a specific action is done.
```

Comment:
What action can that be? Can you give an example here?
Who carries out that action?
I see that you explain a bit of that in the following, but the "entry" is hard to catch up with.

Maybe check these sources to get a better entry into the topic: 
https://developer.mozilla.org/en-US/docs/Web/Events
https://api.jquery.com/category/events/